web/vite.config.ts.timestamp-1695071566216-e680af4895c7b.mjs

```diff
@@ -0,0 +1,22 @@
+// vite.config.ts
+import { defineConfig, loadEnv } from "file:///mnt/d/School/EIP/lastest/erp/web/node_modules/vite/dist/node/index.js";
```

Comment:
you need to remove this file i think

Reply:
erp 185 le telechargement des images au niveau des vulnerabilité

Reply:
je l'ai supprimé cest bon